### PR TITLE
API 32086 Add v2 poa rswag structure

### PR DESCRIPTION
**Pull request by jvcAdHoc**
## Summary

- This pr adds the bones of POA V2 swagger docs

## Related issue(s)
- [API 32086](https://jira.devops.va.gov/browse/API-32086)

## Screenshots
image

image

image

image

image

image

image

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb [hunk 1]
```diff
@@ -129,281 +129,518 @@
     end
   end
 
-  path '/veterans/{veteranId}/power-of-attorney:appoint-individual' do
-    put 'Appoint an individual Power of Attorney for a Veteran.' do
+  # path '/veterans/{veteranId}/power-of-attorney:appoint-individual' do
```
Comment (jvcAdHoc) [inline modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb:132]: commenting these out for now. We don't intend to use the routes defined here, but we might want to reuse some of the specs.

File: modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb [hunk 2]
```diff
@@ -129,281 +129,518 @@
...
+  #
+  #         it 'returns a 422 response' do |example|
+  #           assert_response_matches_metadata(example.metadata)
+  #         end
+  #       end
+  #     end
+  #   end
+  # end
+
+  path '/veterans/{veteranId}/2122a/validate' do
```
Comment (jvcAdHoc) [inline modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb:413]: I used [this pr](https://github.com/department-of-veterans-affairs/vets-api/pull/13355) as an example for making a minimum rswag spec

File: modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb [hunk 3]
```diff
@@ -169,7 +169,7 @@
         }
       end
 
-      describe 'Getting a successful response' do
+      xdescribe 'Getting a successful response', document: false do
```
Comment (jvcAdHoc) [inline modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb:172]: pending the specs that were brought over for endpoint we won't support. We might want to reference and/or delete them later.

**Comment by jvcAdHoc** [comment]
the `/veterans/{veteranId}/power-of-attorney` path was brought over from v1 and I made no changes to it here. We can update the documentation as needed once the v2 update is complete.

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 4]
```diff
@@ -8125,6 +8125,413 @@
...
+                  },
+                  "type": "individual"
+                },
+                "schema": {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "required": [
+                    "code"
+                  ],
+                  "properties": {
```
Comment (acovrig) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:8186]: Should all these (not required) fields be marked `nullable`? In local dev, we saw `null`s for those, not sure if that'd happen in prod though.

**Review by acovrig — APPROVED** [review]

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json (hunk 4 above)
Comment (jvcAdHoc) [reply modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:8249]: I added those

**Review by acovrig — APPROVED** [review]

**Pull request merged**